src/main/java/org/spongepowered/common/world/generation/config/noise/SpongeNoiseConfig.java

```diff
@@ -0,0 +1,198 @@
+        public NoiseConfig.Builder verticalSize(final int vertical) {
+            this.verticalSize = vertical;
+            return this;
+        }
+
+        @Override
+        public NoiseConfig.Builder islandNoiseOverride(boolean islandNoiseOverride) {
+            this.islandNoiseOverride = islandNoiseOverride;
+            return this;
+        }
+
+        @Override
+        public NoiseConfig.Builder largeBiomes(boolean largeBiomes) {
+            this.largeBiomes = largeBiomes;
+            return this;
+        }
+
+        @Override
+        public NoiseConfig.Builder amplified(boolean amplified) {
+            this.amplified = amplified;
+            return this;
+        }
+
+        @Override
+        public NoiseConfig.Builder reset() {
+
+            // defaults like overworld
+            this.minY = -64;
+            this.height = 384;
+            this.sampling = SamplingConfig.of(1D, 80.0D,1D, 160.0D);
+            this.top = SlideConfig.of(-0.078125D, 2, 8);
+            this.bottom = SlideConfig.of(0.1171875D, 3, 0);
+            this.horizontalSize = 1;
+            this.verticalSize = 2;
+            this.islandNoiseOverride = false;
+            this.amplified = false;
+            this.largeBiomes = false;
+            this.terrainShaper = Shaper.overworld();
+            return this;
+        }
+
+        @Override
+        public NoiseConfig.Builder from(final NoiseConfig value) {
+            Objects.requireNonNull(value, "value");
+            this.minY = value.minY();
+            this.height = value.height();
+            this.sampling = value.samplingConfig();
+            this.top = value.topConfig();
+            this.bottom = value.bottomConfig();
+            this.horizontalSize = value.horizontalSize();
+            this.verticalSize = value.verticalSize();
+            this.islandNoiseOverride = value.islandNoiseOverride();
+            this.amplified = value.amplified();
+            this.largeBiomes = value.largeBiomes();
+            this.terrainShaper = value.terrainShaper();
+            return this;
+        }
+
+        @Override
+        public @NonNull NoiseConfig build() {
+            Objects.requireNonNull(this.sampling, "sampling");
+            Objects.requireNonNull(this.top, "top");
+            Objects.requireNonNull(this.bottom, "bottom");
+
+            return (NoiseConfig) (Object) NoiseSettings.create(this.minY, this.height, (NoiseSamplingSettings) this.sampling,
+                    (NoiseSlider) this.top, (NoiseSlider) this.bottom,
+                    this.horizontalSize, this.verticalSize, this.islandNoiseOverride, this.amplified, this.largeBiomes,
+                    (TerrainShaper) (Object) this.terrainShaper);
+        }
+    }
+
+    public static final class FactoryImpl implements NoiseConfig.Factory {
+
+        private static final class Holder {
+            private static final NoiseConfig OVERWORLD = (NoiseConfig) (Object) NoiseSettings.create(-64, 384,
```

Comment:
Boy these constants...can you throw a javadoc line where they come from? I assume you can't use anything in Vanilla itself to target.

Reply:
got them from here: https://github.com/misode/vanilla-worldgen/blob/master/worldgen/noise_settings/overworld.json

Reply:
also available in code - I added comments